libs/application/templates/complaints-to-althingi-ombudsman/src/lib/dataSchema.ts

```diff
@@ -0,0 +1,45 @@
+import * as kennitala from 'kennitala'
+import * as z from 'zod'
+import { error } from './messages/error'
+
+export const ComplaintsToAlthingiOmbudsmanSchema = z.object({
+  approveExternalData: z.boolean().refine((v) => v),
+  externalData: z.object({
```

Comment:
I don't think the externalData becomes part of the application answer schema

Reply:
I will use Application from core. Will add it in the next pull request.